content/tutorials/r_gbif_name_matching/index.Rmd

```diff
@@ -12,70 +12,90 @@ output:
 
-```{r message=FALSE, warning=FALSE}
-library(tidyverse) # tidyverse
-library(rgbif)     # To Match GBIF
-library(inborutils) # wrap GBIF api data
+```{r install_pkgs, eval=FALSE}
+devtools::install_github("ropensci/rgbif")    # install rgbif
+devtools::install_github("inbo/inboruilts")   # install inboruilts
+```
+
+```{r load_libraries, message=FALSE, warning=FALSE}
+library(tidyverse)    # tidyverse
+library(rgbif)        # To Match GBIF
+library(inborutils)   # wrap GBIF API data
 library(knitr)
 ```
 
 ## Read data file containing the scientific names
 
 Read file containing the scientific names you want to check against the [GBIF](https://www.gbif.org/species/search?q=) taxonomic backbone:
 
-```{r}
+```{r read_species_list}
 species_list <- read_csv("https://raw.githubusercontent.com/inbo/inbo-pyutils/master/gbif/gbif_name_match/sample.csv", trim_ws = TRUE, col_types = cols())
 ```
    
 Take a look at the data:
 
-```{r}
-knitr::kable(head(species_list))
+```{r preview_species_list}
+knitr::kable(species_list)
 ```
 
-## Request species information
+## Request taxonomic information
 
-Match the column containing the scientificName with GBIF, using the `gbif_species_name_match` function from the [inborutils](https://inbo.github.io/inborutils/reference/gbif_species_name_match.html) package:
+Match the column containing the scientific name with GBIF, using the `gbif_species_name_match` function from the [inborutils](https://inbo.github.io/inborutils/reference/gbif_species_name_match.html) package:
 
 ```{r, message=FALSE, warning=FALSE}
 species_list_matched <- species_list %>% 
-    gbif_species_name_match(name_col = "name") 
+    gbif_species_name_match(name = "name") 
+```
+
+The `name` argument is the column name containing the scientific names. Default value: `name`. So, the code above is equivalent to:
+
+```{r similar_code, eval=FALSE}
+species_list_matched <- species_list %>% 
+    gbif_species_name_match() 
 ```
 
-The `name_col` argument is the column name containing the scientific names. 
+By default, `gbif_species_name_match` returns the following GBIF fields: `usageKey`, `scientificName`, `rank`, `order`, `matchType`, `phylum`, `kingdom`, `genus`, `class`, `confidence`, `synonym`, `status`, `family`.
 
 Take a look at the updated data:
 
-```{r}
-knitr::kable(head(species_list_matched))
+```{r view_output}
+knitr::kable(species_list_matched)
 ```
 
-You can also specify which info (`gbif_terms`) GBIF should return. For example, return a lot of terms: 
+Notice that GBIF fields whose name is already used as column name are automatically renamed by adding suffix  `1`. In our case, input data.frame `species_list` contains already a column called `kingdom`: the GBIF kingdom values are returned in column `kingdom1`:
+
+```{r zoom_kingdom}
+species_list_matched %>% select(kingdom, kingdom1)
+```
+
+You can specify which GBIF fields you would like to have:
 
 ```{r, message=FALSE, warning=FALSE}
```

Comment:
unnamed chunk. naming all chunks is good practice

Reply:
Indeed. Done here: https://github.com/inbo/tutorials/pull/146/commits/774308dbf8519b2cfc55513eed05106b70ea6dad